engine/src/cmd/enhancement.h

```diff
@@ -7,7 +7,7 @@
 class GameEnhancement : public GameUnit< Enhancement >
 {
     friend class UnitFactory;
-protected:
+public:
 /// constructor only to be called by UnitFactory
```

Comment:
Might want to remove this comment as well

Reply:
Done